client/src/test/java/org/asynchttpclient/request/body/multipart/MultipartUploadTest.java

```diff
@@ -142,6 +157,28 @@ public void sendEmptyFileZeroCopy() throws Exception {
     sendEmptyFile0(false);
   }
 
+  private void sendEmptyFileInputStream0(boolean disableZeroCopy) throws Exception {
+    File file = getClasspathFile("empty.txt");
```

Comment:
Could you please use a non empty file?

Reply:
This helper method is explicitly to test empty file to ensure we're covering the case from issue https://github.com/AsyncHttpClient/async-http-client/issues/1485. I can add an additional test for non-empty file

Reply:
Yes please.

Reply:
Done.